CA-G12/Vet
# 79 delete post route

created ```DELETE /post``` route
test cases included:
- delete a post -- should succeed
- delete a post that wasn't authored by user -- should fail
- delete a post that doesn't exist -- should fail

NOTE: this branch should be added after PR #78
